src/printer.js

```diff
@@ -3002,11 +3002,20 @@ function printArgumentsList(path, options, print) {
       i++;
     }, "arguments");
 
+    const somePrintedArgumentsWillBreak = printedArguments.some(willBreak);
+
     return concat([
-      printedArguments.some(willBreak) ? breakParent : "",
+      somePrintedArgumentsWillBreak ? breakParent : "",
       conditionalGroup(
         [
-          concat(["(", concat(printedExpanded), ")"]),
+          concat([
+            ifBreak(
```

Comment:
I have a feeling this can be simplified to:

```diff
-            ifBreak(
-              indent(concat(["(", softline, concat(printedExpanded)])),
-              concat(["(", concat(printedExpanded)])
-            ),
+            indent(concat(["(", softline, concat(printedExpanded)])),
```

If that doesn't work, what you've got is fine!

Reply:
No, I have tried this before and just tried it again to be sure. It breaks a lot of other tests, adding indentation to functions where it should not.